uwblueprint/paramedics-react
# Email Validation Input Line Jump Fix

[Notion ticket] https://www.notion.so/uwblueprintexecs/396370c5e09f4c28bc7f455a7e84c908?v=3751aa62a2594c74b7e0ecb363304c24&p=11b137cdc90e433181378a6d291fc9d4

[Figma link]

## Changes
- added a wrapper around the email validation field so that the text box will not jump up and down

## Screenshots
![image](https://user-images.githubusercontent.com/44094521/104398412-ac1da000-551c-11eb-9511-b27a5635ada6.png)
![image](https://user-images.githubusercontent.com/44094521/104398426-b17aea80-551c-11eb-84f9-047d17a8b392.png)

## Testing
- go to resource management and add a team member
- type a bad email. The input line should not suddenly jump 

## Checklist
>[Code review doc for reference](https://www.notion.so/uwblueprintexecs/Code-Review-a21fc85b00394f488e92d9d605f6b2bc)

before opening PR

filling out PR

after opening PR
